### DISCUSSION TITLE
Prism plugins command-line

### DISCUSSION THREAD

**Discussion by sebdanielsson** [facebook/docusaurus#4829] [Q&A] [answered]
Hi!
 
I would like to add the following plugin to get a command line for my code blocks. Is this possible with Docusaurus?
https://prismjs.com/plugins/command-line/

Best regards
Sebastian

**Comment by lex111** [comment, marked as answer]
Unfortunately not, because internally we use `prism-react-renderer` to render code blocks, which does not support adding plugins (see https://github.com/FormidableLabs/prism-react-renderer/issues/2 for more details).

**Comment by slorber** [comment]
If the point is just to add a `$` or similar prefix, custom CSS can be good enough. See https://github.com/facebook/docusaurus/discussions/5227#discussioncomment-1082708
